composer.json

```diff
@@ -26,6 +26,9 @@
     "require-dev": {
         "phpunit/phpunit": "^8.4"
     },
+    "suggest": {
```

Comment:
Add `psr/container` to `require` section

Reply:
`psr/container-implementation` should be added as well since console can't work w/o container.